src/components/Toc.tsx

```diff
@@ -1,7 +1,7 @@
 import Link from "next/link";
 
 import { PencilIcon } from "@/icons/Pencil";
-import { ThumpUpIcon } from "@/icons/ThumpUp";
+import { ThumbUpIcon } from "@/icons/ThumbUp";
```

Comment:
Thanks for your contribution! This file will also need to be renamed

Reply:
> Thanks for your contribution! This file will also need to be renamed

https://github.com/inkonchain/docs/pull/255

Done